packages/@n8n/nodes-langchain/utils/descriptions.ts

```diff
@@ -83,7 +83,7 @@ export const promptTypeOptions: INodeProperties = {
 			description: 'Use an expression to reference data in previous nodes or enter static text',
 		},
 	],
-	default: 'auto',
+	default: 'define',
```

Comment:
Needed to change the default here to handle the edge case where we add an `Agent` after a `Manual Trigger`, which automatically includes a `ChatTrigger`.

The logic in the PR ends up seeing two calls to adjust parameters, first from adding both `Agent` and its `ChatTrigger` at the same time - where we'd want to have this field as `auto` - and then later on in the logic when a connection is made from `ManualTrigger` to `Agent`, where we want this as define in a vacuum, but as auto in this case since we'll ultimately have the `ChatTrigger`.

The way this PR solves this case is by setting it to `auto` explicitly, and leaving it as its default `define` otherwise (rather than overwriting the current value).

Reply:
Do we need to version this change? Because we don't save the default value on workflow save but rather populate it on node execution. So if there's an active workflow that had this field with "auto" before, changing it to `define` would break it since the actual prompt would be empty. Or am I missing something?

Reply:
I had assumed this field is only relevant on node creation, thank you for catching that! Will bump the version.